.changeset/short-flowers-fall.md

```
@@ -0,0 +1,13 @@
+'mastra': patch
+'@mastra/client-js': patch
+'@mastra/server': patch
+'@mastra/mongodb': patch
+'@mastra/spanner': patch
+'@mastra/core': patch
+'@mastra/libsql': patch
+'@mastra/mysql': patch
+'@mastra/pg': patch
+---
+
+Add permanent dataset item purging that scrubs SCD-2 history rows, tombstones, and linked experiment result payloads while preserving version and review metadata.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)